spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/DefaultSpanNamer.java

```diff
@@ -28,9 +28,10 @@
  *     <ul>the default provided value</ul>
  * </li>
  *
- * @see org.springframework.cloud.sleuth.SpanName
- *
  * @author Marcin Grzejszczak
+ * @since 1.0.0
```

Comment:
pedantic: usually not necessary to add subminors in since (unless we expect to change api on subminor)